seed/seed.json

```diff
@@ -2438,7 +2438,7 @@
                             "CustomNotificationAds"
                         ]
                     },
-                    "name": "normalized_display_coordinate_x/1/normalized_display_coordinate_y/0/inset_x/0/inset_y/72",
+                    "name": "normalized_display_coordinate_x|1|normalized_display_coordinate_y|0|inset_x|0|inset_y|72",
```

Comment:
`name=value` instead of `name|value`, i.e., `foo=1|bar=2'

Reply:
Fixed